docs/source/references/isi.md

```diff
@@ -0,0 +1,14 @@
+# Iroha Special Instructions
+
+Because Iroha Special Instructions is a very important topic among Iroha functionality we made
+this reference with detailed description of every aspect related to **ISI**.
+
+## TL;DR
+
+In Iroha 2.0 we have Iroha Special Instructions (inspired by commands from Iroha 1 and smart contracts in other systems).
+They are provided out of the box for all legal actions and contain implementations with Permissions check inside. 
```

Comment:
I don't think `legal actions` is a good phrase to use here - I think it's mainly used for discussion of laws, court system, etc.. `for all possible action...` or something like that will be better here,